build-info-api/src/main/java/org/jfrog/build/api/util/Utils.java

```diff
@@ -0,0 +1,35 @@
+package org.jfrog.build.api.util;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ImmutableSetMultimap;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Utils {
+    /**
+     * Build string of properties in the following form:
+     * key=value;key=value...
+     *
+     * @param properties - keys and values to concat
+     * @return
+     */
+    public static String buildPropertiesString(ArrayListMultimap<String, String> properties) {
```

Comment:
I could find at least one more place in the code where this logic is duplicated. See the buildMatrixParamsString method in the DeploymentUrlUtils class.

Reply:
`buildMatrixParams` returns a different result that corrupts the `vcs.url`.

Reply:
buildMatrixParams removed.